modules/core/shared/src/main/scala/lucuma/core/math/units.scala

```diff
@@ -19,6 +19,9 @@ import eu.timepit.refined.types.numeric.PosInt
 import spire.math._
 
 trait units {
+  trait Pixels
```

Comment:
This maybe odd as a unit but I think it is correct. At least it would let us clearly mark what values are in pixels